pylearn2/train_extensions/tests/test_roc_auc.py

```diff
@@ -173,7 +173,7 @@ def test_roc_auc_one_vs_one():
                     max_epochs: 1,
                 },
                 !obj:pylearn2.termination_criteria.MonitorBased {
-                    channel_name: train_y_roc_auc,
+                    channel_name: train_roc_auc-0v1,
```

Comment:
Just curious, if you have time to answer: What happened to train_y_roc_auc? Previously it was there and behaved normally, but after the modifications in train.py it starts to be a Key error. Seems nothing relevant to that is changed in this PR.